source/docs/03-metadata.xml

```diff
@@ -1382,156 +1354,308 @@
                      <specDesc key="relationList"/>
                   </specList>
                </p>
-               <div xml:id="textTitlePages" type="div3">
-                  <head>Title Pages</head>
-                  <p>A specialized element is furnished for the capture of titlepage information.</p>
-                  <p>
-                     <specList>
-                        <specDesc key="titlePage"/>
-                     </specList>
-                  </p>
-                  <p>The <gi scheme="MEI">titlePage</gi> element, modelled after a similar element in Encoded Archival Description (EAD), may occur within the textual matter preceding or following the musical content of the encoding. Since a diplomatic transcription of the titlepage is often necessary to accurately identify musical material contained within a source, <gi scheme="MEI">titlePage</gi> may also be used within the metadata header as a child of the <gi scheme="MEI">physDesc</gi> element.</p>
-                  <p>Detailed analysis of the title page and other preliminaries of older printed books and manuscripts is of major importance in descriptive bibliography and the cataloging of printed books. The following elements are suggested as a means of encoding the major features of most title pages for faithful rendition:</p>
+               <div xml:id="manuscriptsAndPrints" type="div3">
+                  <head>Manuscripts and prints</head>
```

Comment:
You should be consistent about capitalization. Perhaps `Manuscripts and Prints`?

Reply:
Of course! It was solved in a later commit -.- Will be resolved